python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -146,6 +148,13 @@ class RunRequest(
             targets an asset selection, then by default a RunRequest returned from it will launch
             all the assets in the selection. This argument is used to specify that only a subset of
             these assets should be launched, instead of all of them.
+        check_key_selection (Optional[Sequence[AssetCheckKey]]): A subselection of asset checks that
```

Comment:
Sorry for my interminable naming nitpicks, but I think this should either be `asset_check_selection` or `asset_check_keys`:
- Unless it's abundantly clear that we're talking about assets, like in arguments to the `@asset` decorator, we normally say "asset check" instead of "check".
- Including both "key" and "selection" seems redundant.

Between `asset_check_keys` and `asset_check_selection`, I think there's a tradeoff:
- `asset_check_keys` makes it more possible to move to an eventual future where we have `asset_selection: AssetSelection`.
- `asset_check_selection` is more consistent with the current `asset_selection`.

Between those to, I'd lean towards `asset_check_keys` unless you feel strongly otherwise.

Reply:
I'm fine with asset_check_keys.